transcom/mymove
# B 20686 int

## [Agility ticket](tbd)

## Summary

Updating the payment requests tab for TIO. Previously it was required for the TIO to review weights for all payment requests, even if the payment request contained zero service items that were reliant on the weight of the shipments. Now when the payment request ONLY contains some combination of Move Management, Counseling, Crating, or Uncrating the TIO will not be required to review weights prior to approving the payment request.

It does not appear possible to create a payment request for service items that are reliant on weights where the weights have not been entered. So this scenario is specifically for shipments that do NOT contain them, and shouldn't require a weights review.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the milmove application
2. Login as a Customer
3. Create a move that contains at least one shipment, or more if you want to test multiple service items.

4. Access the office application
5. Login as a Service counselor
6. Approve the move details for the move you created

7. Access the office application
8. Approve the move you created

9. Access the prime simulator
10. Create a payment request for the move you created that contains only some combination of Move Management, Counseling, Crating, and Uncrating.

11. Access the application as a TIO
12. Verify you can approve the service items for the move you created without entering weights.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
